polyfills/Array/prototype/@@iterator/polyfill.js

```diff
@@ -0,0 +1,38 @@
+(function () {
+
+// from https://github.com/medikoo/es6-iterator/blob/master/array.js
+'use strict';
+
+var setPrototypeOf = require('es5-ext/object/set-prototype-of')
```

Comment:
This would need to be inlined, we don't use commonJS modules for the bundled polyfills